src/WingetCreateCore/Common/GitHub.cs

```diff
@@ -19,6 +19,7 @@ namespace Microsoft.WingetCreateCore.Common
     using Microsoft.WingetCreateCore.Models.Installer;
     using Octokit;
     using Polly;
+    using AuthenticationType = Octokit.AuthenticationType;

```

Comment:
nit: is this used?

Reply:
I feel it's not needed.

Reply:
Manifest schema `1.10.0` has a new `AuthenticationType` field which conflicts with the definition from `Octokit` library. This is done to disambiguate the reference for the compiler. I could just use `Octokit.AuthenticationType` in multiple places where this reference throughout this file, but I thought this was cleaner to define it up top since we won't be using the installer manifest schema definition in this file

![image](https://github.com/user-attachments/assets/49957dd9-c5a0-403d-86c7-b91183c08fde)